### PR TITLE
Chainsync benchmark improvements.

### PR DESCRIPTION
**Pull request by karknu**

### CODE REVIEW

File: benchmarking/chain-sync/benchmark-chain-sync-mainnet.sh [hunk 1]
```diff
@@ -32,6 +32,8 @@ exec timeout $CPU_TIME_LIMIT ${NODE} \
   --tracing-verbosity-maximal \
   --trace-mempool \
   --trace-forge \
+  --trace-block-fetch-decisions \
+  --trace-chain-db \
```
Comment (dcoutts) [inline benchmarking/chain-sync/benchmark-chain-sync-mainnet.sh]: Can we do exactly the same for the `cardano-node/scripts/mainnet.sh` to fix it there?

Comment (karknu) [reply benchmarking/chain-sync/benchmark-chain-sync-mainnet.sh]: Yes, but that would risk triggering trace/monitoring related space leak.

Comment (CodiePP) [reply benchmarking/chain-sync/benchmark-chain-sync-mainnet.sh]: if that script does not use the "LiveView" then we do not have the tracers added.

Comment (karknu) [reply benchmarking/chain-sync/benchmark-chain-sync-mainnet.sh]: The script uses liveview. The leak seems to be specifically triggered by the `--trace-chain-db` option, which is currently missing.
I'm going to wait with merging this PR and address mainnet.sh until that issue is fixed.

**Comment by karknu** [comment]
This pr has rotten

**Pull request closed without merging**